### PR TITLE
SHS-5053: Consistency: Update to field names

### PR DESCRIPTION
**Pull request by hectorlj**
# READY FOR REVIEW

## Summary
1. Changes `Spotlight` component label and add more label from `Slides` to `Spotlight(s)`
2. Changes `Gradient Hero Link` label to `Link` in the `Banner image with overlay and text` components
## Steps to Test
1. Create a new Flexible Page
2. Add a `Spotlight` component to the `Main Region`
3. Verify the widget label and add more label show `Spotlight(s)` instead of `Slides`
4. Add a ` Banner image(s) with partial overlay and text` component to both the `Main Region` and the `Full-width Region`
5. Add multiple `Banner image with overlay and text` components to each `Banner image(s) with partial overlay and text` component
6. Verify that each instance of `Banner image with overlay and text` link label shows `Link` instead of `Gradient Hero Link`

## PR Checklist
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
- [Humsci Basic PR Checklist](https://github.com/SU-HSDO/suhumsci/blob/develop/docs/HumsciBasicPRChecklist.md)

### CODE REVIEW

**Comment by hectorlj** [comment]
The remaining test doesn't seem to be failing from these code changes
https://app.circleci.com/pipelines/github/SU-HSDO/suhumsci/6341/workflows/4a32cea1-28e1-4e2c-bdc0-ba384c45fc07/jobs/43690/tests

File: docroot/modules/humsci/hs_paragraph_types/hs_paragraph_types.module [hunk 1]
```diff
@@ -39,23 +39,60 @@ function hs_paragraph_types_preprocess_field__field_hs_collection_items(&$variab
...
+      foreach (Element::children($field_widget_form['widget']) as $key) {
+        if (
+          !is_int($key) ||
+          empty($field_widget_form['widget'][$key]['subform']['field_hs_gradient_hero_link']['widget'][0]['#title'])
+        ) {
+          continue;
+        }
+        $field_widget_form['widget'][$key]['subform']['field_hs_gradient_hero_link']['widget'][0]['#title'] = 'Link';
+      }
+      break;
```
Comment (cienvaras) [inline docroot/modules/humsci/hs_paragraph_types/hs_paragraph_types.module]: This paragraph is not used in other components, lets just change the field label in the  configuration, please.

**Review by cienvaras — CHANGES_REQUESTED** [review]
@hectorlj Works as expected, but there's an improvement that can be done to simplify code. Please check the code comments.

File: docroot/modules/humsci/hs_paragraph_types/hs_paragraph_types.module (hunk 1 above)
Comment (hectorlj) [reply docroot/modules/humsci/hs_paragraph_types/hs_paragraph_types.module]: Good catch! Fixed

**Review by cienvaras — APPROVED** [review]
Thanks! :shipit:

**Pull request merged**